src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -1471,9 +1473,16 @@ public abstract class ODataClientTemplate : TemplateBase
                 edmNavigationSourceList.Add(singleton);
             }
         }
-
-        this.WriteGeneratedEdmModel(Utils.SerializeToString(this.context.Edmx).Replace("\"", "\"\""));
-        
+    
+            if(useTempFile) 

```

Comment:
The indentation on this if\else block is off

Reply:
fixed